osgp/platform/osgp-adapter-ws-publiclighting/src/main/java/org/opensmartgridplatform/adapter/ws/publiclighting/application/services/NotificationServicePublicLighting.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2019 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.ws.publiclighting.application.services;
+
+import org.opensmartgridplatform.adapter.ws.clients.SendNotificationServiceClient;
+import org.opensmartgridplatform.adapter.ws.schema.core.common.OsgpResultType;
+import org.opensmartgridplatform.adapter.ws.schema.core.notification.Notification;
+import org.opensmartgridplatform.adapter.ws.schema.core.notification.NotificationType;
+import org.opensmartgridplatform.adapter.ws.shared.services.NotificationService;
+import org.opensmartgridplatform.shared.exceptionhandling.WebServiceSecurityException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.util.StringUtils;
+
+public class NotificationServicePublicLighting implements NotificationService {
```

Comment:
Consider renaming to PublicLightingNotificationService

Reply:
Done